cocos/scene-graph/node.ts

```diff
@@ -1472,8 +1475,8 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
     protected _dirtyFlags = TransformBit.NONE; // does the world transform need to update?
     protected _eulerDirty = false;
 
-    protected _flagChangeVersion = 0;
-    protected _hasChangedFlags = 0;
+    // The high bits are used to store the version number of the changedFlag, and the low 3 bits represent its specific value
+    protected _versionedChangedFlags = 0;
```

Comment:
- please use a more meaningful name
- and please add more detail information about the bits meaning, for example,
```
____________________
| 31 - 3: xxx |  2-0: yyy|
____________________
```

Reply:
fixed